docs/symfony-integration.md

```diff
@@ -164,13 +175,14 @@ if (SamplingResult::RECORD_AND_SAMPLED === $samplingResult->getDecision()) {
     ->getTracer('io.opentelemetry.contrib.php');
 
     $request = Request::createFromGlobals();
-    $jaegerSpan = $jaegerTracer->startAndActivateSpan($request->getUri());
-    $zipkinSpan = $zipkinTracer->startAndActivateSpan($request->getUri());
-
+    $jaegerSpan = $jaegerTracer->spanBuilder($request->getUri())->startSpan();
+    $jaegerSpan->activate();
+    $zipkinSpan = $zipkinTracer->spanBuilder($request->getUri())->startSpan();
```

Comment:
Should use only one `TracerProvider` / `Span`.
Also should detach the scope returned by `Span::activate()` when the span is ended.

Reply:
Thanks! Updated